packages/syntax-core/src/SelectList/SelectList.module.css

```diff
@@ -32,11 +32,6 @@
   margin: 0;
 }
 
-.selectBox:focus-visible {
```

Comment:
@JFChoe `SelectList` now shows a blue border when it's selected with the mouse. I'm not sure if that's what we want.

![image](https://user-images.githubusercontent.com/127199/232162093-60aa31e8-6e75-45ab-9fba-79f941a963ec.png)

Reply:
thought that was original behavior for some reason. fixing rn

Reply:
Ah, yeah, we do show a border on click but this is not the right color. Should be `Primary700`, visual reference here: https://www.figma.com/file/p7LKna9JMU0JEkcKamzs53/%F0%9F%93%90-Syntax?node-id=1201-8095&t=2DlXndqjgyf8V7Mb-11

Reply:
Alright I just added in a video to show the behavior to the description. Lmk if it looks good and/or want to see in person  @ajmooo !